lib/src/method_channel.dart

```
@@ -43,32 +46,45 @@ class ShowMenuArgs {
   }
 }
 
-const _channel = MethodChannel('native_context_menu');
+final _channel = const MethodChannel('native_context_menu')
+  ..setMethodCallHandler(
+    (call) async {
+      switch (call.method) {
+        case 'id':
+          {
+            // Notify about the selected menu item's id & complete the future.
+            _completer.complete(call.arguments);
+            break;
+          }
+        default:
+          break;
```

should throw an exception